gpep517/__main__.py

```diff
@@ -322,6 +331,12 @@ def add_build_args(parser):
                        help="Patch sysconfig paths to use specified sysroot "
                             "(experimental cross-compilation support)",
                        type=Path)
+    group.add_argument("--rewrite-prefix-from",
+                       help="Additional prefix applied to the build host "
+                            "(only applicable with --sysroot option)")
+    group.add_argument("--rewrite-prefix-to",
+                       help="Additional prefix applied to the target host "
+                            "(only applicable with --sysroot option)")
```

Comment:
Ideally, could you also add an explicit error if either of these is set without `--sysroot`? If not, I'll do it later.